clang/test/Driver/sycl-spirv-ext-old-model.c

```diff
@@ -129,3 +186,110 @@
 // CHECK-CPU-SAME:,+SPV_KHR_non_semantic_info
 // CHECK-CPU-SAME:,+SPV_KHR_cooperative_matrix
 // CHECK-CPU-SAME:,+SPV_INTEL_fp_max_error"
+//
+// CHECK-CPU-NFPME: llvm-spirv{{.*}}"-spirv-allow-unknown-intrinsics=llvm.genx.,llvm.fpbuiltin"
+// CHECK-CPU-NFPME-SAME: {{.*}}"-spirv-ext=-all
+// CHECK-CPU-NFPME-SAME:,+SPV_EXT_shader_atomic_float_add
+// CHECK-CPU-NFPME-SAME:,+SPV_EXT_shader_atomic_float_min_max
+// CHECK-CPU-NFPME-SAME:,+SPV_KHR_no_integer_wrap_decoration,+SPV_KHR_float_controls
+// CHECK-CPU-NFPME-SAME:,+SPV_KHR_expect_assume,+SPV_KHR_linkonce_odr
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_subgroups,+SPV_INTEL_media_block_io
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_device_side_avc_motion_estimation
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_fpga_loop_controls
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_unstructured_loop_controls,+SPV_INTEL_fpga_reg
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_blocking_pipes,+SPV_INTEL_function_pointers
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_kernel_attributes,+SPV_INTEL_io_pipes
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_inline_assembly,+SPV_INTEL_arbitrary_precision_integers
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_float_controls2
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_vector_compute,+SPV_INTEL_fast_composite
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_arbitrary_precision_fixed_point
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_arbitrary_precision_floating_point
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_variable_length_array,+SPV_INTEL_fp_fast_math_mode
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_long_constant_composite
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_arithmetic_fence
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_cache_controls
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_fpga_buffer_location
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_fpga_argument_interfaces
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_fpga_invocation_pipelining_attributes
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_fpga_latency_control
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_task_sequence
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_token_type
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_bfloat16_conversion
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_joint_matrix
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_hw_thread_queries
+// CHECK-CPU-NFPME-SAME:,+SPV_KHR_uniform_group_instructions
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_masked_gather_scatter
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_tensor_float32_conversion
+// CHECK-CPU-NFPME-SAME:,+SPV_INTEL_optnone
+// CHECK-CPU-NFPME-SAME:,+SPV_KHR_non_semantic_info
+// CHECK-CPU-NFPME-SAME:,+SPV_KHR_cooperative_matrix
+// CHECK-CPU-NFPME-NOT:,+SPV_INTEL_fp_max_error"
```

Comment:
```suggestion
// CHECK-CPU-NFPME-NOT:,+SPV_INTEL_fp_max_error"
```
To match up with the `-check-prefixes` suggestion above.